src/audio/src/src.c

```diff
@@ -64,20 +67,20 @@
 LOG_MODULE_REGISTER(src, CONFIG_SOF_LOG_LEVEL);
 
 /* Calculates the needed FIR delay line length */
-static int src_fir_delay_length(struct src_stage *s)
+int src_fir_delay_length(struct src_stage *s)
```

Comment:
I'm guessing we need to expose these now to common SRC code ? If so can we move the small functions that we want to inline to the header and make static inline (and force inlining) especially for any function we call during ```process()```

Reply:
I moved small functions to src.h as static inline.